addons/hr_holidays/models/hr_leave_allocation.py

```diff
@@ -214,6 +216,28 @@ def _update_accrual(self):
 
             holiday.write(values)
 
+    def _compute_accrual_plan(self):
+        accrual_plans=self.env['hr.accrual.plan'].search([("company_id","=",self.env.company.id)])
```

Comment:
Actually, you should define an multi company ir.rule.

1/ The restriction on the evironment company would be implicit.
2/ This could be seen as a security leak to make people access to HR policy from another company.

+ Linting issue.

Reply:
There is an issue here. Depending on the environment company means that you'll make different computation, and provide different holidays durations according to the company on which you're logged into...

You should take the time off type company as reference, which is not contextual.